packages/client/src/lib/theme.js

```diff
@@ -10,4 +10,11 @@ export const theme = createTheme({
   typography: {
     fontFamily: "'Inter', sans-serif",
   },
+  devices: {
+    mobile: `(max-width: 374px)`,
+    sm_tablet: `(max-width: 499px)`,
+    tablet: `(max-width: 799px)`,
+    laptop: `(max-width: 1023px)`,
+    desktop: `(max-width: 1439px)`,
```

Comment:
Shall we go with 1920 and 1440 that we have in design? 🤔

Reply:
Forgot about that 😃